docs/index.md

```diff
@@ -33,39 +36,33 @@ Discover new tools to help:
 
 - [Virtual Environments](wiki-guide/Virtual-Environments.md): Summary of `conda` and `pip` environments: how to make, use, and share them.
 
-
-## Collaborative Infrastructure We Use
+## Collaborative Infrastructure used by Imageomics and ABC
 
 - GitHub
-    - [Institute Code Repositories](https://github.com/Imageomics) where we store our code (software + tools).
+    - [Imageomics](https://github.com/Imageomics) and [ABC](https://github.com/ABC-Center) Code Repositories, where we store our code (software + tools).
     - GitHub's [Docs](https://docs.github.com/en)
         - [Repositories](https://docs.github.com/en/repositories)
         - [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects)
 - Hugging Face
-    - [Imageomics Organization page](https://huggingface.co/imageomics) where we store our datasets and models (and their metadata).
+    - [Imageomics](https://huggingface.co/imageomics) and [ABC](https://hf.co/ABC-Center) Organization Pages, where we store our datasets and models (and their metadata).
     - Additionally, use [Hugging Face Spaces](https://huggingface.co/docs/hub/spaces) to run demos of models and other projects.
     - Hugging Face's [Docs](https://huggingface.co/docs)
         - [Model Hub](https://huggingface.co/docs/hub/models-the-hub)
         - [Datasets](https://huggingface.co/docs/hub/datasets-overview)
 
 ### Collaborative Infrastructure Diagram
-![tech_infrastructure_diagram](wiki-guide/images/index/382108831-1173cd79-db94-4326-8b6e-dcbdeb8939cd.png)
 
+![tech_infrastructure_diagram](wiki-guide/images/index/collaborative-infrastructure-diagram.png)
 
-## Imageomics Branding (Logos)
+## `<Organization>` Branding (Logos)
 
-We have two versions of the logo, a [fish](logos/Imageomics_logo_fish.png) and a [butterfly](logos/Imageomics_logo_butterfly.png), which should be used for scientific posters, conference, workshop, and meeting marketing materials, etc. Choice of logo is based on user preference.
-
-![butterfly logo](logos/Imageomics_logo_butterfly.png){: style="width:45%"}
-![fish logo](logos/Imageomics_logo_fish.png){: style="width:45%"}
+Since this is a joint Imageomics and ABC project, we have a special combined logo:
 
+![combined Imageomics and ABC logo](logos/Imageomics_ABC.png){: style="width:75%"}
 
 ## Other pages of note
+
 - [Glossary for Imageomics](wiki-guide/Glossary-for-Imageomics.md): Collection of terms used in imageomics. The goal is to ensure all participating domains are represented, thus facilitating interdisciplinary communication. This is a group effort, please check it out and add terms you think should be there!
 - [Command Line Cheat Sheet](wiki-guide/Command-Line-Cheat-Sheet.md): Collection of useful bash and git commands with some git tips.
 
-
-<br>
-<br>
-
```

Comment:
I removed the two `<br>` tags here to satisfy markdownlint rule **MD033** (no inline HTML). Relying on `<br>` for spacing mixes presentation into content and can render inconsistently across themes.

If it's important to maintain this space above the footer, we could inject bottom whitespace via CSS:

1. **Create a file** `docs/stylesheets/extra.css` with something like:

   ```css
   /* For Material for MkDocs */
   .md-typeset {
     padding-bottom: 2rem;
   }
   /* For default MkDocs theme */
   .content {
     padding-bottom: 2rem;
   }
   ```

2. **Enable it** in the `mkdocs.yml`:

   ```yaml
   extra_css:
     - stylesheets/extra.css
   ```

This keeps the Markdown clean, avoids inline HTML, and gives consistent vertical space before the footer.